Whiskey/Functions/Get-WhiskeyPSModule.ps1

```diff
@@ -52,90 +52,102 @@ function Get-WhiskeyPSModule
 
     Set-StrictMode -Version 'Latest'
     Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
-    
+
+    Write-WhiskeyDebug '\Get-WhiskeyPSModule\' -Indent
+
     Register-WhiskeyPSModulePath -PSModulesRoot $PSModulesRoot
-    
-    Write-WhiskeyDebug '[Get-WhiskeyPSModules]  PSModulePath:'
-    $env:PSModulePath -split [IO.Path]::PathSeparator |
-        ForEach-Object { "  $($_)"} |
-        Write-WhiskeyDebug
-    $modules = Get-Module -Name $Name -ListAvailable -ErrorAction Ignore
-    $modules | Out-String | Write-WhiskeyDebug
-    $modules |
-        Where-Object {
-            if( -not $Version )
-            {
-                return $true
-            }
-
-            $moduleInfo = $_
-
-            $moduleVersion = $moduleInfo.Version
-            $prerelease = ''
-            if( ($moduleInfo | Get-Member 'PreRelease') )
-            {
-                $prerelease = $moduleInfo.PreRelease
-            }
-            else
-            {
-                $privateData = $moduleInfo.PrivateData
-                if( $privateData )
+
+    try
+    {
+        Write-WhiskeyDebug '\env:PSModulePath\' -Indent

```

Comment:
Yes. Future enhancement. I didn't go that far because this was supposed to be a small fix and was taking longer than anticipated and I've been noodling over a generic logging module where this kind of functionality would go.